src/components/beneficiaries/BeneficiariesFormComponent.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+    VOLUNTEER_FORM,
+    VOLUNTEER_INITAL_VALUES,
+    VOLUNTEER_SCHEMA,
+} from "./beneficiariesFormData";
+import FormikComponent from "../formik/FormikComponent";
+
+function BeneficiariesFormComponent() {
+    /**
+     * Gets the form field data from the back end
+     * TODO: complete the function
+     */
+    const getData = () => {
+        return {
+            form: VOLUNTEER_FORM,
+            initialValues: VOLUNTEER_INITAL_VALUES,
+            schema: VOLUNTEER_SCHEMA,
+        };
+    };
+
+    const { form: formSections, initialValues, schema } = getData();
+
+    return (
```

Comment:
Absolutely right, 
I think by adding another property in returned object from getData() method will just work good. 
    const getData = () => {
        return {
            **name**: 'Beneficiaries form',
            form: BENEFICIARIES_FORM,
            initialValues: BENEFICIARIES_INITAL_VALUES,
            schema: BENEFICIARIES_SCHEMA,
        };
    }; 

but if we want real impact then we need to update FormikComponent to accept one more prop named (name)

Reply:
I think having a title/description object would be a good idea, since then it can be used to generate the page as you said. But I think it should not be passed to `FormikComponent` since it's not part of the form, it can just be consumed by the `BeneficiariesFormComponent`?

Reply:
If we pass it to FormikComponent then we can name our html form element.